evap/staff/forms.py

```diff
@@ -238,7 +238,9 @@ def __init__(self, *args, **kwargs):
             disable_all_fields(self)
 
         if self.instance.pk:
-            self.fields['responsible'].initial = self.instance.responsible_contributors[0]
+            responsible = self.instance.responsible_contributors[0]
+            self.fields['responsible'].queryset = UserProfile.objects.filter(Q(pk=responsible.pk) | Q(is_active=True))
```

Comment:
maybe `self.fields['responsible'].queryset = self.fields['responsible'].queryset | UserProfile.objects.filter(pk=responsible.pk)` if that works? i think that communicates the intent (use the existing queryset but also include the current choice) more clearly.

maybe the |= operator works?

Reply:
i tried that before but it didn't work as expected. will try again, though.

Reply:
don't know what the problem was before, it works with `|=` ... just perfectly